Sources/Extensions/UIViewController.swift

```diff
@@ -29,8 +29,8 @@ extension UIViewController {
     }
 
     func openUrlInApp(_ url: URL, showAlert: Bool? = nil) {
-        let vc = SFSafariViewController(url: url, entersReaderIfAvailable: false)
-        present(vc, animated: true)
+        let SFSafariVC = SFSafariViewController(url: url, entersReaderIfAvailable: false)
```

Comment:
Personally for local context. I would just call controller but a variable name cannot with an uppercase letter. We should keep the camel case for a name

Reply:
So I should rename the variable SFSafariVC to sfSafariVC ?

Reply:
To keep consistence inside the projet. Just sfSafariVC.